Jenkinsfile

```diff
@@ -123,6 +123,18 @@ pipeline {
             sh './gradlew --no-daemon itest'
           }
         }
+        stage('blackbox tests') {
+          agent { label 'medium' }
+          tools {
+            jdk 'jdk11'
+          }
+          steps {
+            sh 'git clean -xdff'
+            checkout scm
+            sh 'git submodule update --init'
+            sh './gradlew --no-daemon hdfsTest monitoringTest gtest'
```

Comment:
is there a good reason not to bundle them together with itest (as it used to be in the dsl repo)?

Reply:
Yes, we already run itest in separate stages with different jdks